benchmarks/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.ballerinalang</groupId>
         <artifactId>ballerina-parent</artifactId>
-        <version>0.970.0-alpha5-SNAPSHOT</version>
+        <version>0.970.0-beta2-SNAPSHOT</version>
```

Comment:
We need to add this module to default profile or wso2-release profile? Otherwise, version will not be incremented with each release

Reply:
Yes. It must be a part of Ballerina release build. We can run benchmarks using a different profile.

Reply:
if we add to wso2-release profile ,  then we need to build with the  -Dwarmup.iterations=<No_of_warmUp_Iterations> -Dbenchmark.iterations=<No_of_benchmark_Iterations>